src/module/Markdown/src/Markdown/Options/ModuleOptions.php

```diff
@@ -9,7 +9,9 @@
  */
 namespace Markdown\Options;
 
-class ModuleOptions
+use Zend\Stdlib\AbstractOptions;
+
+class ModuleOptions extends AbstractOptions
```

Comment:
This is a helper for injecting configuration options into the object. If it's missing, the class will not be able to properly ingest the configuration settings. I basically forgot to add it and was wondering for quite a while why I was unable to change the config of this module.